src/intrinsics/Promise.mts

```diff
@@ -212,6 +213,189 @@ function* Promise_all([iterable = Value.undefined]: Arguments, { thisValue }: Fu
   return result;
 }
 
+/** https://tc39.es/proposal-await-dictionary/#sec-promise.allkeyed */
+function* Promise_allKeyed([promises = Value.undefined]: Arguments, { thisValue }: FunctionCallContext): ValueEvaluator {
+  // 1. Let C be the this value.
```

Comment:
Hi! I would prefer to remove spec steps if they're very straitforward.

For example I will keep comments like this:

```js
// 1. If an implementation-defined debugging facility is available and enabled, then
```

```js
  // 1. If StringToCodePoints(text) is not a valid JSON text as specified in ECMA-404, throw a SyntaxError exception.
  Q(JSONValidator.validate(text));
```